SimPEG/Examples/MT_1D_analytic_nlayer_Earth.py

```diff
@@ -0,0 +1,428 @@
+from scipy.constants import epsilon_0, mu_0
+import matplotlib.pyplot as plt
+import numpy as np
+from ipywidgets import *
+from SimPEG.EM.Utils import k, omega
+
+"""
+MT1D: n layered earth problem
+*****************************
+
+Author: Thibaut Astic
+Contact: thast@eos.ubc.ca
+Date: January 2016
```

Comment:
git keeps track of all of this: odds are the example will be updated - so I would suggest removing this so that we don't have to manually keep track of it